openpype/hosts/maya/plugins/publish/validate_review.py

```diff
@@ -0,0 +1,30 @@
+import pyblish.api
+
+from openpype.pipeline.publish import (
+    ValidateContentsOrder, KnownPublishError
+)
+
+
+class ValidateReview(pyblish.api.InstancePlugin):
+    """Validate review."""
+
+    order = ValidateContentsOrder
+    label = "Validate Review"
+    families = ["review"]
+
+    def process(self, instance):
+        cameras = instance.data["cameras"]
+
+        # validate required settings
+        if len(cameras) == 0:
+            raise KnownPublishError(
+                "No camera found in review instance: {}".format(instance)
+            )
+        elif len(cameras) > 2:
+            raise KnownPublishError(
+                "Only a single camera is allowed for a review instance but "
+                "more than one camera found in review instance: {}. "
+                "Cameras found: {}".format(instance, ", ".join(cameras))
+            )
```

Comment:
These should be `PublishValidationError` I believe. Right @iLLiCiTiT @mkolar ? Or would those be influenced by design changes for the new publisher in the coming days?

Reply:
Yes, should be `PublishValidationError`. `KnownPublishError` would stop publishing.